packages/ui/client/composables/summary.ts

```diff
@@ -43,6 +43,11 @@ export const time = computed(() => {
   if (t > 1000)
     return `${(t / 1000).toFixed(2)}s`
 
+  // Can occur when usage of timer mocking. 
+  // TODO: we should isolate that timer mocking to avoid that case.
```

Comment:
maybe we should do just that instead of small patching? 👀

Reply:
I'm not confortable with timer mocking, but yes, you're right! Is someone able to work on a fix?